src/inc/msquic.h

```diff
@@ -1161,6 +1161,7 @@ typedef enum QUIC_CONNECTION_EVENT_TYPE {
 #ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
     QUIC_CONNECTION_EVENT_RELIABLE_RESET_NEGOTIATED         = 16,   // Only indicated if QUIC_SETTINGS.ReliableResetEnabled is TRUE.
     QUIC_CONNECTION_EVENT_ONE_WAY_DELAY_NEGOTIATED          = 17,   // Only indicated if QUIC_SETTINGS.OneWayDelayEnabled is TRUE.
+    QUIC_CONNECTION_EVENT_DATA_ACKED                        = 18,   
```

Comment:
I wouldn't call this `DATA_ACKED`. Maybe `NETWORK_STATISTICS`

Reply:
Well, the event triggers upon data acked. What's being returned is net stats. It felt a little strange to call it a 'network stats' event. Do we anticipate someone using this event as a pure indication of data being acknowledged to do something other than network stats with it?

Reply:
We might end up triggered for other things too

Reply:
Ok fair, can change.